docker/datahub-gms/Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM openjdk:8 as builder
 COPY . /datahub-src
 RUN cd /datahub-src && ./gradlew :gms:war:build \
-    && cp gms/war/build/libs/war.war /gms.war
+    && cp gms/war/build/libs/war.war /war.war
```

Comment:
Why renaming?

Reply:
Why was there renaming in the first place (the original cp was a rename + move)? This makes mounting easier.

Reply:
Got it.